topics/migration.md

```diff
@@ -0,0 +1,307 @@
+| 7.0   | 8.0    |
+
+## Migrate a standalone instance
+
+To migrate a standalone Redis server to Valkey, you have the following options:
+
+* [Physical migration](#physical-migration) by copying the most recent on-disk snapshot from the Redis server to Valkey and starting Valkey server with it
+* [Setting up replication](#repication) between Redis and Valkey 
+* [Migrating specific keys](#migrate-specific-keys) 
+
+The example migration steps are provided for Redis 7.2.5 and Valkey version 7.2.6. 
+
+### Physical migration
+
+This is the easiest and fastest migration method. You make a fresh snapshot of your Redis instance and copy it over to Valkey. Valkey reads the data from the snapshot on startup and restores its contents into memory. The tradeoff for this method is the downtime to shutdown Redis and wait for Valkey to load the data. 
+
+1. Make a backup of your Redis instance. 
+
+    1. Connect to Redis and check the number of keys:
+
+        ```
+        $ redis-cli -h 127.0.0.1 -p 6379
+        127.0.0.1:6379> INFO KEYSPACE
+        # Keyspace
+        db0:keys=6286,expires=0,avg_ttl=0
+        ```
+    
+    2. Check the configuration for the directory (`dir`) where Redis stores its database files and the name of the database file (`dbfilename`):
+
+        ```
+        redis> CONFIG GET dir dbfilename
+        1) "dir"
+        2) "/data"
+        3) "dbfilename"
+        4) "dump.rdb"
+        ```
+    
+    3. Check the timestamp of the last save operation
+
+        ```
+        redis> LASTSAVE
+        (integer) 1724764878
+        ```
+
+    4. Start a backup
+
+        ```
+        redis> BGSAVE
+        Background saving started
+        ```
+
+    5. Check if the backup succeeded by running LASTSAVE periodically. The backup ended when the timestamp changes.
+
+2. Stop the Redis server. 
+3. Copy the RDB file to the Valkey's data directory and start Valkey. 
+
+   >NOTE: If you enabled AOF in your Valkey configuration,  disable it on the first start. Otherwise, the copied RDB file will not be imported into Valkey.
+
+   For Docker deployments, copy the RDB file to your host and start a Valkey container mounting this file to the container's data directory, 
+
+    ```
+    docker cp <container-name>:/data/dump.rdb <path/on/host>
+    ```
+
+    Start Valkey:
+
+    ```
+    docker run -d --name somevalkey -v <path/on/host>:/data valkey/valkey
+    ```
+
+4. Check the keyspace on Valkey to verify that the data is migrated:
+
+    ```
+    docker exec -it somevalkey valkey-cli
+    valkey> INFO KEYSPACE
+    # Keyspace
+    db0:keys=6286,expires=0,avg_ttl=0
+    ```
+
+### Replication
+
+To minimize the downtime during migration, you can use replication - a native feature of Redis / Valkey that allows replaying data on another server to handle the workload.
+
+In this scenario we will configure Valkey to be the replica of Redis. Both Redis and Valkey are running in separate Docker containers connected to the same network. 
```

Comment:
The use of Docker in this example makes it seem like you _need_ Docker to do this. Maybe indicate that you're using docker for illustrative purposes?